ftl/python/builder.py

```diff
@@ -36,11 +36,17 @@
 class Python(builder.JustApp):
     def __init__(self, ctx):
         self.descriptor_files = [_REQUIREMENTS_TXT]
+        # TODO(aaron-prindle) map  python_version to interpreter_parser
+        # will prob need to pass in args to the builder
+        self.interpreter_parser = lambda: 'python2.7'
+        self.pkg_txt_parser = ftl_util.descriptor_parser  # detect phase here
+        self.pkg_key_parser = ftl_util.descriptor_parser
         self.namespace = _PYTHON_NAMESPACE
-        self.ctx = ctx
-        self._tmp_app = self._gen_tmp_dir(_TMP_APP)
+        # self.ctx = ctx
```

Comment:
Why are these all commented out?

Reply:
they should needed with the new refactor and also _ctx if provided by the base class so ctx can be removed.  They are removed now